src/components/Document/Paragraph.tsx

```diff
@@ -5,7 +5,12 @@ export function Paragraph(props: {
 	className?: string;
 }) {
 	return (
-		<p className={cn("text-base leading-8 text-f-200 mb-3", props.className)}>
+		<p
+			className={cn(
+				"text-base leading-relaxed text-f-200 mb-3",
+				props.className,
+			)}
+		>
 			{props.children}
```

Comment:
This line height feels too tight, needs more breathing room here.  

<img width="871" alt="image" src="https://github.com/user-attachments/assets/88995745-7d6b-4c2c-afbd-91099e87ee9d">

Reply:
i think it was way to wide before, this feels good to me. docs should be a bit more dense imo